apps/console/src/features/users/components/bulk-import-response-list.tsx

```diff
@@ -249,13 +301,65 @@ export const BulkImportResponseList: React.FunctionComponent<BulkImportResponseL
         );
     };
 
+    /**
+     * Handles the flow mode switch.
+     *
+     * @param _ - Event.
+     * @param newTabIndex - New tab index.
+     */
+    const handleTabChange = (_: React.SyntheticEvent, newTabIndex: number): void => {
+        bulkResponseOperationTypes.map((item: DropdownItemProps) => {
+            if (item.key === newTabIndex) {
+                setResponseOperationType(item.value as BulkImportResponseOperationTypes);
+                setResponseOperationTypeTab(newTabIndex);
+            }
+        });
+    };
+
+    /**
+     * Render user creation and user assignment operation type selection.
+     */
+    const resolveBulkResponseOperationTypeSelection = (): ReactElement => {
+        return(
+            <Tabs
+                value={ responseOperationTypeTab }
+                onChange={ handleTabChange }
+                aria-label="basic tabs example"
+            >
+                { bulkResponseOperationTypes.map(( item: DropdownItemProps) => (
+                    <Tab
+                        key={ item.key }
+                        label={ (
+                            <div className="beta-feature-tab-item">
+                                <Typography sx={ { fontWeight: 500 } }>{ item.text }</Typography>
```

Comment:
Let's avoid inline styles

Reply:
@PasinduYeshan Whats the conclusion here?

Reply:
This approach had been used in other places [1](https://github.com/wso2/identity-apps/blame/b965bcd84751f71c0a1762a3cb50a924214d2b41/features/admin.authentication-flow-builder.v1/components/authentication-flow-builder.tsx#L391). Due to the urgency, I might have missed this. This needs to be fixed.

Reply:
resolve this comment, and add the issue which you were mentioning